{
  "id": "oval:org.opensuse.security:def:202145452",
  "version": "1",
  "class": "vulnerability",
  "metadata": {
    "Title": "CVE-2021-45452",
    "affected": {
      "family": "unix",
      "Platform": [
        "openSUSE Tumbleweed"
      ]
    },
    "reference": [
      {
        "ref_id": "Mitre CVE-2021-45452",
        "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45452",
        "source": "CVE"
      },
      {
        "ref_id": "SUSE CVE-2021-45452",
        "ref_url": "https://www.suse.com/security/cve/CVE-2021-45452",
        "source": "SUSE CVE"
      },
      {
        "ref_id": "SUSE-SU-2022:0102-1",
        "ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2022-January/010010.html",
        "source": "SUSE-SU"
      },
      {
        "ref_id": "SUSE-SU-2022:0103-1",
        "ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2022-January/010016.html",
        "source": "SUSE-SU"
      },
      {
        "ref_id": "SUSE-SU-2022:0286-1",
        "ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2022-February/010163.html",
        "source": "SUSE-SU"
      },
      {
        "ref_id": "openSUSE-SU-2023:0005-1",
        "ref_url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/UHF5IZKTZ2T4T4QQYZMUFHW422X3WCU6/",
        "source": "SUSE-SU"
      }
    ],
    "Description": "\n    Storage.save in Django 2.2 before 2.2.26, 3.2 before 3.2.11, and 4.0 before 4.0.1 allows directory traversal if crafted filenames are directly passed to it.\n    ",
    "advisory": {
      "from": "security@suse.de",
      "Severity": "Moderate",
      "cve": {
        "text": "CVE-2021-45452",
        "href": "https://www.suse.com/security/cve/CVE-2021-45452/",
        "impact": "moderate",
        "cvss3": "5.3/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N"
      },
      "bugzilla": [
        {
          "text": "SUSE bug 1194116",
          "href": "https://bugzilla.suse.com/1194116"
        }
      ],
      "issued": {
        "date": "2022-01-12"
      },
      "updated": {
        "date": "2023-01-04"
      },
      "affected_cpe_list": {
        "Cpe": [
          "cpe:/o:opensuse:tumbleweed"
        ]
      }
    }
  },
  "criteria": {
    "operator": "AND",
    "criterias": [
      {
        "operator": "OR",
        "criterions": [
          {
            "test_ref": "oval:org.opensuse.security:tst:2009663231",
            "comment": "python310-Django-4.0.1-1.1 is installed"
          },
          {
            "test_ref": "oval:org.opensuse.security:tst:2009663232",
            "comment": "python38-Django-4.0.1-1.1 is installed"
          },
          {
            "test_ref": "oval:org.opensuse.security:tst:2009663233",
            "comment": "python39-Django-4.0.1-1.1 is installed"
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:org.opensuse.security:tst:2009634834",
        "comment": "openSUSE Tumbleweed is installed"
      }
    ]
  }
}
